oshi-core/src/main/java/oshi/hardware/platform/linux/LinuxSoundCard.java

```diff
@@ -57,10 +56,14 @@ public LinuxSoundCard(String kernelVersion, String name, String codec) {
         private static List<File> getCardFolders() {
                 File cardsDirectory = new File(SC_PATH);
                 List<File> cardFolders = new ArrayList<>();
-                for (File card : Objects.requireNonNull(cardsDirectory.listFiles())) {
-                        if (card.getName().startsWith(CARD_FOLDER) && card.isDirectory()) {
-                                cardFolders.add(card);
+                try {
+                        for (File card : cardsDirectory.listFiles()) {
+                                if (card.getName().startsWith(CARD_FOLDER) && card.isDirectory()) {
+                                        cardFolders.add(card);
+                                }
                         }
+                }catch (Exception e){
```

Comment:
Travis builds now!  I hope this was just a debugging step to confirm, because catching all exceptions here is a bad idea and even catching NPE's that you could avoid by explicitly checking null seems like a code smell.  Can we just do:
```
foo = cardsDirectory.listFiles();
if (foo != null) { for (File card : foo) { ... } }
```

Reply:
Yes I did this for testing travis ci .
In response to your code suggestion on the else statement I just simply log it like "error: no files found! " or something like that and return the empty list ?
Sorry for this as it is a bit tedious to debug considering the only way to see if it works is by committing and waiting for travis's results.

Reply:
Thought it might be a test. No worries.  

A log message (warm level or lower) seems fine for the case where there are no audio cards.